### PR TITLE
changes after updating 2-moment distribution parameters

### PR DESCRIPTION
**Pull request by crocicc**
## Content

----
- [ ] I have read and checked the items on the review checklist.

### CODE REVIEW

File: src/Microphysics2M.jl [hunk 1]
```diff
@@ -445,27 +446,34 @@ function rain_evaporation(
...
 
-    - `struct` - type for 2-moment rain autoconversion parameterization
-    - `q_liq` - cloud water specific humidity
+    - `structs` - structs with SB2006 cloud droplets and raindrops 
+                size distributions parameters
+    - `q_liq` - cloud water specific humidity 
     - `q_rai` - rain water specific humidity
     - `N_liq` - cloud droplet number density
     - `N_rai` - rain droplet number density
+    - `N_liq` - cloud droplet number density
```
Comment (trontrytel) [inline src/Microphysics2M.jl:461]: Looks like during rebasing we accepted both lines. We only need one `N_liq` etc :)

File: src/Microphysics2M.jl [hunk 2]
```diff
@@ -494,45 +499,57 @@ function radar_reflectivity(
...
 
-    - `struct` - type for 2-moment rain autoconversion parameterization
-    - `q_liq` - cloud water specific humidity
+    - `structs` - structs with SB2006 cloud droplets and raindrops 
+                size distribution parameters
+    - `q_liq` - cloud water specific humidity 
     - `q_rai` - rain water specific humidity
     - `N_liq` - cloud droplet number density
     - `N_rai` - rain droplet number density
+    - `N_liq` - cloud droplet number density
```
Comment (trontrytel) [inline src/Microphysics2M.jl:538]: Same here, the arguments are repeated

File: src/Microphysics2M.jl [hunk 3]
```diff
@@ -561,17 +575,22 @@ function effective_radius(
...
 
     Bc =
         (xc == 0) ? FT(0) :
-        (SF.gamma(FT(νc + 1)) * xc / SF.gamma(FT(νc + 2)))^(-1)
+        ((SF.gamma(FT(νc + 1) / μc) * xc / SF.gamma(FT(νc + 2) / μc))^(-μc))
     Br =
         (xr == 0) ? FT(0) :
         ((SF.gamma(FT(νr + 1) / μr) * xr / SF.gamma(FT(νr + 2) / μr))^(-μr))
-    Ac = (N_liq * Bc^(FT(νc + 1))) / SF.gamma(FT(νc + 1))
+    Ac = (μc * N_liq * Bc^(FT(νc + 1) / μc)) / SF.gamma(FT(νc + 1) / μc)
```
Comment (trontrytel) [inline src/Microphysics2M.jl:593]:
```suggestion
    Ac = μc * N_liq * Bc^(FT(νc + 1) / μc) / SF.gamma(FT(νc + 1) / μc)
```

File: src/Microphysics2M.jl [hunk 4]
```diff
@@ -561,17 +575,22 @@ function effective_radius(
...
+    q_rai = (q_rai < τ_q) ? FT(0) : q_rai
+
     xc = (N_liq < τ_N) ? FT(0) : ((q_liq * ρ_air) / N_liq)
     xr = (N_rai < τ_N) ? FT(0) : ((q_rai * ρ_air) / N_rai)
     C = FT((4 / 3) * π * ρ_w)
 
     Bc =
         (xc == 0) ? FT(0) :
-        (SF.gamma(FT(νc + 1)) * xc / SF.gamma(FT(νc + 2)))^(-1)
+        ((SF.gamma(FT(νc + 1) / μc) * xc / SF.gamma(FT(νc + 2) / μc))^(-μc))
```
Comment (trontrytel) [inline src/Microphysics2M.jl:589]:
```suggestion
        (SF.gamma(FT(νc + 1) / μc) * xc / SF.gamma(FT(νc + 2) / μc))^(-μc)
```

File: src/Microphysics2M.jl [hunk 5]
```diff
@@ -586,13 +605,17 @@ function effective_radius(
...
+"""
 """
     effective_radius_Liu_Hallet_97(q_liq, q_rai, N_liq, N_rai, ρ_air, ρ_w)
 
+    - `q_liq` - cloud water specific humidity
     - `q_liq` - cloud water specific humidity
     - `q_rai` - rain water specific humidity
     - `N_liq` - cloud droplet number density
     - `N_rai` - rain droplet number density
+    - `N_liq` - cloud droplet number density
```
Comment (trontrytel) [inline src/Microphysics2M.jl:617]: same here

File: src/parameters/Microphysics2M.jl [hunk 6]
```diff
@@ -375,7 +381,32 @@ function ParticlePDF_SB2006(td::CP.AbstractTOMLDict)
...
+"""
+    CloudParticlePDF_SB2006
+
+Cloud droplets size distribution parameters from SB2006
+
+# Fields
+$(DocStringExtensions.FIELDS)
+"""
+Base.@kwdef struct CloudParticlePDF_SB2006{FT} <: ParametersType{FT}
+    "Raindrop size distribution coefficient νc"
```
Comment (trontrytel) [inline src/parameters/Microphysics2M.jl:396]:
```suggestion
    "Cloud droplet size distribution coefficient νc"
```

File: src/parameters/Microphysics2M.jl [hunk 7]
```diff
@@ -375,7 +381,32 @@ function ParticlePDF_SB2006(td::CP.AbstractTOMLDict)
...
+    νc::FT
+    "Raindrop size distribution coefficient μc"
```
Comment (trontrytel) [inline src/parameters/Microphysics2M.jl:398]:
```suggestion
    "Cloud droplet size distribution coefficient μc"
```

File: src/Microphysics2M.jl [hunk 8]
```diff
@@ -494,45 +499,57 @@ function radar_reflectivity(
...
+
     xc = (N_liq < τ_N) ? FT(0) : ((q_liq * ρ_air) / N_liq)
     xr = (N_rai < τ_N) ? FT(0) : ((q_rai * ρ_air) / N_rai)
     C = FT(4 / 3 * π * ρ_w)
     Z₀ = FT(1e-18)
 
     Bc =
         (xc == 0) ? FT(0) :
-        (SF.gamma(FT(νc + 1)) * xc / SF.gamma(FT(νc + 2)))^FT(-1)
+        ((SF.gamma(FT(νc + 1) / μc) * xc / SF.gamma(FT(νc + 2) / μc))^(-μc))
```
Comment (trontrytel) [inline src/Microphysics2M.jl:514]:
```suggestion
        (SF.gamma(FT(νc + 1) / μc) * xc / SF.gamma(FT(νc + 2) / μc))^(-μc)
```

File: docs/src/Microphysics2M.md [hunk 9]
```diff
@@ -116,7 +116,7 @@ The default free parameter values are:
 |``\nu``     | ``2``                                             |
 |``A``       | ``400``                                           |
 |``a``       | ``0.7``                                           |
-|``c``       | ``3``                                             |
+|``b``       | ``3``                                             |
```
[the remark this answers is not in the document]
Comment (crocicc) [reply docs/src/Microphysics2M.md:119]: There was an error in the documentation. Following the SB2006 paper and ClimaParams, `c=4` is the accretion parameter, while in the autoconversion formula we have two parameters `a` and `b`, which are equal to 0.7 and 3. In the formula in line 106, `b` and `c` must be replaced with `a` and `b` respectively. I just change it in the last commit.

[the remark this answers is not in the document]
Comment (trontrytel) [reply docs/src/Microphysics2M.md:119]: Thank you!

**Pull request merged**